src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/PrefsFramework.md

```diff
@@ -0,0 +1,775 @@
+
+The value is an object containing the properties of the preference.
+Every preference in a Primary Schema must have at least two properties: `“type”` and `“default”`.
+
+_Coming soon: More information about these two properties_
+
+#### Panel ####
+
+
+A [Panel](../Panels.md) is a [component](../UnderstandingInfusionComponents.md) responsible for rendering the user interface controls for a
+preference and tying them to the internal model] that represents the preference value.
+The Panel for the auto-pilot preference control is defined in the `minEditor.js` file:
+<aside class="infusion-docs-callout">
+[Components](../UnderstandingInfusionComponents.md) are the core building-blocks of
+any Infusion application.
+An Infusion component can represent a visible component on screen, a collection of related
+functionality (such as an object, as in object-orientation), or simply a unit of work or
+relationship between other components.
+</aside>
+
+```javascript
+/**
+ * Panel for the auto-pilot preference
+ */
+fluid.defaults("minEditor.panels.autoPilot", {
+    gradeNames: ["fluid.prefs.panel"],
+
+    // the Preference Map maps the information in the primary schema to this panel
+    preferenceMap: {
+        // the key must match the name of the pref in the primary schema
+        "minEditor.autoPilot": {
+            // this key is the path into the panel's model where this preference is stored
+            "model.autoPilot": "default"
+        }
+    },
+
+    // selectors identify elements in the DOM that need to be accessed by the code;
+    // in this case, the Renderer will render data into these particular elements
+    selectors: {
+        autoPilot: ".mec-autoPilot"
+    },
+
+    // the ProtoTree is basically instructions to the Renderer
+    // the keys in the prototree match the selectors above
+    protoTree: {
+        // this value is an IoC reference to the last part of the model path in the preferenceMap
+        autoPilot: "${autoPilot}"
+    }
+});
+```
+In this code snippet, the Panel is created using a call to the Infusion Framework function
+`fluid.defaults()`, just as the Primary Schema was. As with the Primary Schema, the call to
+`fluid.defaults()` is passed two arguments: 1) a string name (`"minEditor.panels.autoPilot"`), and
+2) a JavaScript object containing options for configuring the component – in this case, the Panel.
+
+The screenshot in Figure 2 (above) shows what the Panel looks like to the user: A single checkbox
+with a label, with a header above. The options for configuring this Panel
+include four properties: `gradeNames`, `preferenceMap`, `selectors` and `prototree`:
+<dl>
+<dt>`gradeNames`</dt>
+<dd>As we saw with the Primary Schema, any call to `fluid.defaults()` must include the `gradeNames`
+property in the options argument. Panels must use the `"fluid.prefs.panel"` grade.
+Using this grade automatically buys you a lot of Framework supports necessary for Panels.</dd>
```

Comment:
"buys you a lot of Framework supports necessary for Panels" might be a good spot for an aside to list out some of those supports.